components/order_manager.py

```diff
@@ -22,8 +23,8 @@ def fetch_orders(self, order_id=None):
 
         if order_id:
             order = db_ref.document(str(order_id)).get()
-            return vars(order).get("_data")
-
+            data = vars(order).get("_data")
```

Comment:
if order is None, then vars(order) will break.

Reply:
no it does not break

Reply:
i have tested it